igc_lib.py

```diff
@@ -10,15 +10,12 @@
 
 def sphere_distance(lat1, lon1, lat2, lon2):
     """Computes the great circle distance on a unit sphere.
-
```

Comment:
Those empty lines were intentional. I tried to follow Google's Python Style Guide:
https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments